.cirrus.yml

```diff
@@ -43,10 +43,10 @@ task:
     set -xe
 
     # Download the bitcoind binary
-    curl -O https://bitcoincore.org/bin/bitcoin-core-24.1/bitcoin-24.1-x86_64-linux-gnu.tar.gz
-    echo "c112af3d19cca7f5b5f942708ca4d522110d8e1dc1c0f8d3077f531c94e5f00f bitcoin-24.1-x86_64-linux-gnu.tar.gz" | sha256sum -c
-    tar -xzf bitcoin-24.1-x86_64-linux-gnu.tar.gz
-    export BITCOIND_PATH=bitcoin-24.1/bin/bitcoind
+    curl -O https://bitcoincore.org/bin/bitcoin-core-25.0/bitcoin-25.0-x86_64-linux-gnu.tar.gz
+    echo "5df67cf42ca3b9a0c38cdafec5bbb517da5b58d251f32c8d2a47511f9be1ebc2 bitcoin-25.0-x86_64-linux-gnu.tar.gz" | sha256sum -c
```

Comment:
```
+ echo '5df67cf42ca3b9a0c38cdafec5bbb517da5b58d251f32c8d2a47511f9be1ebc2 bitcoin-25.0-x86_64-linux-gnu.tar.gz'
+ sha256sum -c
bitcoin-25.0-x86_64-linux-gnu.tar.gz: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match
```

Reply:
Fixed sorry. Was the shasum for the source tarball.